content/docs/09-registries-and-packs/4.5-deploy-pack.md

```diff
@@ -0,0 +1,938 @@
+  <br />
+
+  ```yaml
+  pack:
+      namespace: "hello-universe"
+  manifests:
+      hello-universe:
+          registry: ghcr.io
+          repository: spectrocloud/hello-universe
+          tag: 1.0.12
+  ```
+
+  <br />
+
+  You can optionally define *presets*, which are predefined values to use in the **values.yaml**. You define presets in a separate **presets.yaml** file. The presets become available when you create the cluster profile. Presets facilitate configuring the profile and avoid errors that can happen by manually editing the **values.yaml** file. Refer [Pack Presets](/registries-and-packs/pack-constraints#packpresets) for details and examples of how to define presets. 
+  
+  The example below shows the parameters you can configure in the **values.yaml** for the `hello-universe` manifest when you create the cluster profile. 
+
+  <br />
+
+  ![Screenshot of the configurable parameters in the values.yaml file.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_profile-values-yaml.png )
+
+  <br />
+
+* **manifests** -  This directory contains the manifest files for your Kubernetes application. This tutorial has only one file, **hello-universe.yaml**. Note that the **values.yaml** file has a corresponding `manifests/hello-universe` element with the same name as the YAML file. 
+  <br />
+
+* **logo.png** -  This file contains a logo that displays when you create a cluster profile. 
+<br />
+
+
+* **README.md** -  This file may contain the pack description, purpose, authors, and other relevant information. The README in the current example introduces the application used in the pack.
+<br />
+
+
+After finalizing all files in the pack directory, the next step is to set up a registry server and publish the pack to that registry, where you can access it directly from Palette. 
+
+<br />
+
+# Set Up the Registry Server
+
+The tutorials environment already has the Spectro registry service and other necessary tools available. The following sections will guide you to start the registry server, expose the service to the external world using [Ngrok](https://ngrok.com/) reverse proxy, and log in to the registry server to push your custom add-on pack to it.   
+
+## Start and Expose the Registry Server
+Start the registry server by issuing the following command from the bash session you opened into the tutorials container. 
+<br />
+
+```bash
+registry serve /etc/spectro/config.yml > /var/log/registry.log 2>&1 &
+```
+
+The registry server will start in HTTP mode (not HTTPS). Refer to the [Add a Custom Registry](/registries-and-packs/adding-a-custom-registry) guide to learn more about deploying an HTTPS registry server. 
+
+
+Next, expose the registry server to the public so that you can configure it later in Palette. Use Ngrok reverse proxy to expose the registry server listening on port 5000 via an HTTP tunnel using the following command. 
```

Comment:
```suggestion
Next, expose the registry server to the public so that you can configure it later in Palette. Use Ngrok reverse proxy to expose the registry server listening on port 5000 via an HTTP tunnel by invoking the following command. 
```